upgrade.php

```diff
@@ -198,6 +201,10 @@ function url_get_contents ($Url) {
 
         }
 
+        if ($env_key == 'MAIL_ENCRYPTION') {
+            $env_good .= '! Your .env still contains MAIL_ENCRYPTION (set to '.$env_value.'). This setting has been deprecated. You might need to add MAIL_TLS_VERIFY_PEER=false to your .env to make your mail work as expected.'."\n";
+        }
+
```

Comment:
I think if we're going to recommend a value for MAIL_TLS_VERIFY_PEER, we should look at the value of MAIL_ENCRYPTION to make sure it makes sense..... if MAIL_ENCRYPTION=tls, they probably _don't_ want MAIL_TLS_VERIFY_PEER=false.